worlds/stardew_valley/data/craftable_data.py

```diff
@@ -26,11 +29,13 @@ class CraftingRecipe:
     item: str
     ingredients: Dict[str, int]
     source: RecipeSource
+    mod_name: Optional[str] = None

```

Comment:
Don't set a default value here, if there is already a default value in the constructor.